other/concat-exports.js

```diff
@@ -0,0 +1,88 @@
+const appendToFile = require('fs').appendFile
+const resolve = require('path').resolve
+const htmlTagNames = require('html-tag-names')
+const svgTagNames = require('svg-tag-names')
+
+function capitalize(s) {
+  return s.slice(0, 1).toUpperCase() + s.slice(1)
+}
+
+function dashToCamelCase(s) {
+  return s.replace(/-([a-z])/g, ([, char]) => char.toUpperCase())
+}
+
+const jsBuiltInObjectNames = [
+  'Array',
+  'ArrayBuffer',
+  'AsyncFunction',
+  'Atomics',
+  'Boolean',
+  'DataView',
+  'Date',
+  'Error',
+  'EvalError',
+  'Float32Array',
+  'Float64Array',
+  'Function',
+  'Generator',
+  'GeneratorFunction',
+  'Infinity',
+  'Int16Array',
+  'Int32Array',
+  'Int8Array',
+  'InternalError',
+  'Intl',
+  'Iterator',
+  'JSON',
+  'Map',
+  'Math',
+  'NaN',
+  'Number',
+  'Object',
+  'ParallelArray',
+  'Promise',
+  'Proxy',
+  'RangeError',
+  'ReferenceError',
+  'Reflect',
+  'RegExp',
+  'SIMD',
+  'Set',
+  'SharedArrayBuffer',
+  'StopIteration',
+  'String',
+  'Symbol',
+  'SyntaxError',
+  'TypeError',
+  'TypedArray',
+  'URIError',
+  'Uint16Array',
+  'Uint32Array',
+  'Uint8Array',
+  'Uint8ClampedArray',
+  'WeakMap',
+  'WeakSet',
+  'WebAssembly',
+]
+
+function unCollide(name) {
+  const prefix = 'html'
```

Comment:
I chose `html` because users may already be familiar with this prefix since it is used in JavaScript (`htmlFor`). But since component names have to be capitalized (otherwise React/JSX treats the tag as custom element), I think postfix `Tag` (ie `ObjectTag`) is a nicer alternative. What do you think?